### DISCUSSION TITLE
Como vem a conquista do tubarão?

### DISCUSSION THREAD

**Discussion by brsdolol** [community/community#20682] [Other Feature Feedback, Questions, & Ideas] [answered]
Eu realmente não sei do que se trata essa conquista, mas alguém sabe ao certo como obtê-la?

**Comment by Kayke-Fujinaka** [comment, marked as answer]
Hi, my friend!
Pull Shark check your PR record
x1 - over 2 PR
x2 - over 16 PR
x3 - over 128 PR
x4 - over 1024 PR

You can create new branch on your repository, code something.... and merge it to main.
Do this 2 times and you will get the badge x1

**Reply by brsdolol** [reply]
THX

**Comment by sheilalarmour** [comment]
Just raise your hand.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

From: ***@***.***>
Sent: Saturday, July 16, 2022 9:09 AM
To: ***@***.***>
Cc: ***@***.***>
Subject: [GitHub Community] How come Shark Achievement? (Discussion #20682)


I don't really know what that achievement is about, but does anyone know for sure how to get it?
